script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -571,42 +628,48 @@ Right now, all of it is directed toward me.
 [sha:00ad87144e7c78d266b34011cbcfe0ed9a46a1e7]{
 -- Page 459, Offset 28350.
 -- ノエルに気付かれないよう、袖の下にナイフを忍ばせる。
-I hide my knife under my sleeve, being careful not to be noticed by Noel.
+I tuck my knife into my sleeve,
+ careful not to let Noel notice as I do.
```

Comment:
Hmm, not sure I agree. I thought this was more he sticks his hand in his pocked and transfers the knife to his sleeve so he can whip it out later. I feel it would be very hard practically to brandish a knife but also keep it up your sleeve.

Reply:
It just feels like the way the clauses are built, it's more like he saying he did the second clause because of the first, i.e, hid his  knife as to not get noticed by her

Reply:
Hmm, yeah you're right. I scanned back and there's no previous clues about him having it in his hand or anything, so I'd assumed it was previously in his pocket (and therefore still concealed?) and he was transferring it to the _inside_ of his sleeve / up his sleeve so that he could palm it quickly. The only later references we get are 

> 指先が震えて、**忍ばせていたナイフを取り落としそうになる**。

and then a bit after that

> ナイフを握り締めて<女|ノエル>を睨む。

which kinda suggests it's now in his hand? So maybe this first line him grabbing it, then holding it in his hand but at a weird angle so the blade is hidden behind (but _outside of_) his sleeve? Unclear where exactly 下 is in this context. I guess just be vague. 

> I hide my knife behind my sleeve so that Noel doesn't notice I'm holding it.